CHANGELOG.md

```diff
@@ -10,3 +10,19 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Added
 * Set up project and CI/CD GitHub actions, [PR-9](https://github.com/reductstore/reduct-go/pull/9)
 * Implement Bucket API [PR-10](https://github.com/reductstore/reduct-go/pull/10)
+* Implement Query API with streaming support for large records
```

Comment:
the detailed list must be in the PR description. Here we we put only one sentence with the link to the PR.